src/directive/sort-by.directive.ts

```diff
@@ -27,16 +31,24 @@ export class JhiSortByDirective implements AfterViewInit {
 
     @Input() jhiSortBy: string;
 
-    sortAscIcon = 'fa-sort-up';
-    sortDescIcon = 'fa-sort-down';
+    sortIcon = faSort;
```

Comment:
Is this needed here, since we are defining them inside the constructor too right?

Reply:
No, it’s the same for the icon. I don’t want destroy the old code! It’s just a copy.